doc/classes/VisualServer.xml

```diff
@@ -124,6 +124,16 @@
 				If [code]true[/code], preserves the horizontal aspect ratio which is equivalent to [constant Camera.KEEP_WIDTH]. If [code]false[/code], preserves the vertical aspect ratio which is equivalent to [constant Camera.KEEP_HEIGHT].
 			</description>
 		</method>
+		<method name="camera_transform_physics_interpolation">
+			<return type="void" />
+			<argument index="0" name="camera" type="RID" />
+			<argument index="1" name="transform" type="Transform" />
+			<description>
+				Transforms both the current and previous stored transform for a camera.
+				This allows transforming a camera without creating a "glitch" in the interpolation.
+				This is particularly useful for large worlds utilising a shifting origin.
```

Comment:
Each linebreak creates a new paragraph (which takes space), so I think this makes more sense as a single paragraph:

```suggestion
				Transforms both the current and previous stored transform for a camera. This allows transforming a camera without creating a "glitch" in the interpolation. This is particularly useful for large worlds utilising a shifting origin.
```